src/gui/deletionconfirmationdialog.ui

```diff
@@ -44,7 +44,7 @@
         </sizepolicy>
        </property>
        <property name="text">
-        <string notr="true">deletion message goes here</string>
+        <string notr="true">message goes here</string>
```

Comment:
Did you forget the `removal` word here?

Reply:
This is some place holder text to the developers, it won't be shown to the users and I intended this way.